### DISCUSSION TITLE
"Failed to setup Minecraft"This error occurred when I first built the environment.

### DISCUSSION THREAD

**Discussion by littleFlyBoy** [FabricMC/fabric-api#2362] [Mod Dev Support] [answered]
![image](https://user-images.githubusercontent.com/76845387/175865698-bb76317b-cf8f-49c3-9a0f-44ff8383b6c1.png)

**Comment by warjort** [comment, marked as answer]
You appear to have a corrupted file? Downloaded from here: https://piston-meta.mojang.com/mc/game/version_manifest.json
Likely it is the cached version in your gradle cache that is broken. 
It is located at ````.gradle/caches/fabric-loom/version_manifest.json```` in your home folder. If you delete it, it should redownload it?

**Reply by littleFlyBoy** [reply]
ok It's work.Thank you.
